### PR TITLE
[2.10.0] System Requirements

### PR DESCRIPTION
**Pull request by michaelstingl**
@fmoc please double-check

### CODE REVIEW

**Review by mmattel — APPROVED** [review]

**Comment by mmattel** [comment]
@mstingl, thanks for the PR.
Please file PR's into master and tell to where a backport is needed to - which would be in this case 2.10.

@fmoc waiting for your ok to proceed

File: modules/ROOT/pages/installing.adoc [hunk 1]
```diff
@@ -27,16 +27,16 @@ macOS::
 (x86-64 or Apple M in Rosetta 2 emulation; unsupported legacy builds for Mac OS X 10.10 & 10.11 available)
 
 Linux::
-* CentOS 7.6+ & 8 (x86-64)
+* CentOS 7.6+x (x86-64)
```
Comment (fmoc) [inline modules/ROOT/pages/installing.adoc:30]: Not sure what `x` stands for. But we've dropped 8.

File: modules/ROOT/pages/installing.adoc [hunk 2]
```diff
@@ -27,16 +27,16 @@ macOS::
...
 +
 [source,console]
 ----
 sudo yum install epel-release
 ----
 * Debian 10 & 11 (x86-64)
-* Fedora 33 & 34 & 35 (x86-64)
-* openSUSE Leap 15.2 & 15.3 (x86-64)
+* Fedora 34 & 35 (x86-64)
+* openSUSE Leap 15.3 (x86-64)
```
Comment (fmoc) [inline modules/ROOT/pages/installing.adoc]: We also support 15.2 for this release, but since it's EOL it will be removed in the next release.

**Review by fmoc — APPROVED** [review]

**Comment by michaelstingl** [comment]
> Please file PR's into master

Good point. Will do…

File: modules/ROOT/pages/installing.adoc (hunk 1 above)
Comment (michaelstingl) [reply modules/ROOT/pages/installing.adoc:30]: idea for better notation?

**Comment by mmattel** [comment]
> CentOS 7.6+x (x86-64)

from that what I understand is, that all versions of 7.6 upwards are supported but not 8

Proposing: `CentOS 7.x with _minimum_ version x=6 (x86-64)`

**Comment by fmoc** [comment]
Looks good to me.

File: modules/ROOT/pages/installing.adoc [hunk 3]
```diff
@@ -27,15 +27,15 @@ macOS::
 (x86-64 or Apple M in Rosetta 2 emulation; unsupported legacy builds for Mac OS X 10.10 & 10.11 available)
 
 Linux::
-* CentOS 7.6+ & 8 (x86-64)
+* CentOS 7.6+x (x86-64)
```
Comment (mmattel) [inline modules/ROOT/pages/installing.adoc:30]:
```suggestion
* CentOS 7.x with _minimum_ version x=6 (x86-64)
```

**Pull request merged**